functional/agent_UUID_assignment_options/test.sh

```diff
@@ -49,15 +49,18 @@ rlJournalStart
         rlRun "limeStopAgent"
     rlPhaseEnd
 
+    # disable agent_UUID_assignment_options test regarding to bz#2165055
+    if [ -n "$UUID_PHASE" ]; then
     rlPhaseStartTest "Generate UUID"
         AGENT_ID="generate"
         rlRun "limeUpdateConf agent  uuid '\"$AGENT_ID\"'"
         rlRun "limeStartAgent"
         sleep 3
-        GENERATE_UUID=$(systemctl status keylime_agent -all | tail -2 | head -1 | grep -oP '(?<=Agent )[^ ]*')
+        GENERATE_UUID=$(journalctl _SYSTEMD_INVOCATION_ID=`systemctl show -p InvocationID --value keylime_agent.service` | grep -oP '(?<=Generated a new UUID: ).*')
```

Comment:
please use
```
$( systemctl show -p InvocationID --value keylime_agent.service) 
```
instead of
```
`systemctl show -p InvocationID --value keylime_agent.service`
```
as it is better readable when commands are nested.